File: Praktikum 4/py/PRAK404_2410817110012_GtMuhammadLuthfiKamil.py
def kalkulator():
    nama = input("Masukkan nama Anda : ")

    while True:
        print("Pilih program")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Pembagian")
        print("5. Exit")

        pilihan = int(input("Masukkan Pilihan :"))
        if pilihan == 5:
            print(f"Terimakasih, telah menggunakan kalkulator {nama}")
            break
        elif pilihan in[1, 2, 3, 4]:
            nilai1 = float(input("Masukkan nilai pertama :"))
            nilai2 = float(input("Masukkan nilai kedua :"))

            if pilihan == 1:
                hasil = nilai1 + nilai2
                print(f"Hasil penjumlahan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f} \n")
            elif pilihan == 2:
                hasil = nilai1 - nilai2
                print(f"Hasil Pengurangan antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f} \n")
            elif pilihan == 3:
                hasil = nilai1 * nilai2
                print(f"Hasil Perkalian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f} \n")
            elif pilihan == 4:
                hasil = nilai1 / nilai2
                print(f"Hasil Pembagian antara {nilai1:.2f} dengan {nilai2:.2f} adalah {hasil:.2f} \n")
        else:
            print("Input anda salah, silahkan coba lagi \n")
kalkulator()

File: Praktikum 4/c/PRAK404_2410817110012_GtMuhammadLuthfiKamil.c
#include <stdio.h>

int main() {
    int mau;
    float bil1, bil2, hasil;
    char nama[100];

    printf("Masukkan nama Anda : ");
    fgets(nama, sizeof(nama), stdin);

    do {
        printf("Pilihan program \n");
        printf("1. Penjumlahan \n");
        printf("2. Pengurangan \n");
        printf("3. Perkalian \n");
        printf("4. Pembagian \n");
        printf("5. Exit \n");
        printf("Masukkan Pilihan : ");
        scanf("%d", &mau);

        if(mau >= 1 && mau <= 4) {
            printf("Masukkan nilai pertama : ");
            scanf("%f", &bil1);
            printf("Masukkan nilai kedua : ");
            scanf("%f", &bil2);

            switch(mau) {
                case 1:
                hasil = bil1 + bil2;
                printf("Hasil Penjumlahan antara %.2f dengan %.2f adalah %.2f \n", bil1, bil2, hasil);
                break;
                case 2:
                hasil = bil1 - bil2;
                printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f \n", bil1, bil2, hasil);
                break;
                case 3:
                hasil = bil1 * bil2;
                printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f \n", bil1, bil2, hasil);
                break;
                case 4:
                hasil = bil1 / bil2;
                printf("Hasil pembagian antara%.2f dengan %.2f adalah %.2f \n", bil1, bil2, hasil);
                break;
            }
        }
        else if (mau == 5) {
            printf("Terimakasih, telah menggunakan kalkulator %s", nama);
        }
        else {
            printf("Input Anda salah, silahkan coba lagi \n");
        }
        printf("\n");
    }
    while(mau != 5);
    return 0;
}